spec/systems/search_keywords_spec.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'search keywords', type: :system do
+  ActiveJob::Base.queue_adapter = :test
```

Comment:
🤔

Reply:
Fixed in `feature/google-scraping` branch

Reply:
Fixed in edc8d731ef828877d299da5a6db6889cb41d859e